econet_base.yaml

```diff
@@ -224,3 +250,82 @@ button:
       - switch.turn_on: alhisclr
       - delay: 1s
       - switch.turn_off: alhisclr
+number:
+  - platform: econet
+    name: "IRTC Year"
+    id: irtc_year
+    number_datapoint: IRTCYEAR
+    unit_of_measurement: "y"
```

Comment:
What's the point of having all these settings when all these are internal?

Reply:
I assume you're referring to:
```yaml
  entity_category: "config"
  icon: mdi:calendar-clock
  unit_of_measurement: "y"
```

I guess those only effect client-side display in HASS. I was originally pushing them to HASS, but then made them internal. I don't think there is much downside to including them, and they will result in cleaner display if users remove the `internal` flag for some usecase.

Reply:
I'm sort of torn as exposing them as a bunch of separate values is kind of janky. You can already set them with the Write service call (which is how I set the time on my HPWH) and that _should_ be a one time operation. My hope was to confirm these showed up for all device types and help people debug their time.

I had also considered adding a button to "set water heater to current time" or the like.